notebooks/contrib-dev/McCabe_Thiele_Steven_Final.ipynb

```diff
@@ -0,0 +1,1665 @@
+          },
+          "metadata": {}
+        }
+      ],
+      "source": [
+        "liqvapy = liq_vap_data[\"vapor mole frac methanol\"]\n",
+        "liqvapx = liq_vap_data[\"liquid mole frac methanol\"]\n",
+        "\n",
+        "# Feature matrix (store in 'X')\n",
+        "X = np.column_stack([np.ones(len(liqvapx))] + [liqvapx**i for i in range(1, 7)])\n",
```

Comment:
Move this to inside the function. It makes life easier for the user. Also, please add comments to explain this better. For example, show the user what a row of the feature matrix looks like in the math before this code.